src/main/webapp/WEB-INF/index.html

```diff
@@ -1,22 +1,60 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <meta charset="UTF-8">
-    <title>Tasks Scheduler</title>
-    <link href="/style.css" rel="stylesheet">
+  <meta charset="UTF-8">
+  <title>Tasks Scheduler</title>
+  <link rel="stylesheet" href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,400italic|Roboto+Mono:400,500|Material+Icons">
+  <link rel="stylesheet" href="daterangepicker.css">
+  <link rel="stylesheet" href="style.css">
```

Comment:
Please keep `/` at the beginning. For now the difference is not visible, because the whole application is under one path fragment at most, but if you'd later add a page at url e.g. `DOMAIN/tasks/mine`, then:
- href="/style.css" would request `DOMAIN/style.css` - good
- href="style.css" would request `DOMAIN/tasks/style.css` - probably not what you wanted